### PR TITLE
notice type filtering

### PR DESCRIPTION
**Pull request by mcoughlin**
This PR tries to better handle notice type filtering.

### CODE REVIEW

**Review by Theodlz — APPROVED** [review]
LGTM

File: skyportal/facility_apis/ztf.py [hunk 1]
```diff
@@ -1191,20 +1191,21 @@ def fetch_depot_observations(instrument_id, session, depot_url, jd_start, jd_end
...
-    obstable.rename(
-        columns={
-            'obsjd': 'obstime',
-            'maglim': 'limmag',
-            'fwhm': 'seeing',  # equivalent as plate scale is 1"/pixel
-        },
-        inplace=True,
-    )
-    obstable['target_name'] = None
+    if len(dfs) > 0:
```
Comment (Theodlz) [inline skyportal/facility_apis/ztf.py:1194]: Someone has been looking at Sentry

File: alembic/versions/0f1f62adc91c_notice_types.py [hunk 2]
```diff
@@ -0,0 +1,337 @@
...
+            'HAWC_BURST_MONITOR',
+            'AMON_NU_EM_COINC',
+            'ICECUBE_ASTROTRACK_GOLD',
+            'ICECUBE_ASTROTRACK_BRONZE',
+            'SK_SN',
+            'ICECUBE_CASCADE',
+            name='noticetype',
+        ),
+        type_=sa.String(),
+        existing_nullable=True,
```
Comment (Theodlz) [inline alembic/versions/0f1f62adc91c_notice_types.py:174]: What is this migration doing? I get from the models that you don't use the ENUM from pygcn anymore, so why does it not just drop the enum here? Just curious

File: skyportal/models/gcn.py [hunk 3]
```diff
@@ -12,7 +12,6 @@
 import json
 import random
 
-import gcn
```
Comment (Theodlz) [inline skyportal/models/gcn.py:15]: GOOD RIDDANCE.

**Review by Theodlz — COMMENTED** [review]
Just one question about the migration

File: alembic/versions/0f1f62adc91c_notice_types.py (hunk 2 above)
Comment (mcoughlin) [reply alembic/versions/0f1f62adc91c_notice_types.py:174]: But there is still a column. So it's an alter I guess.

**Pull request merged**